### PR TITLE
INTLY-3169 -  Update amq online version to be GA image tag version

### PR DESCRIPTION
**Pull request by KevFan**
## Description
The AMQ online CSV already currently uses the GA image, however the versioning does not match the image tag. In addition, the image does not seem to point to the latest 1.1 patch tag according to the [registry](https://access.redhat.com/containers/?tab=overview#/registry.access.redhat.com/amq7/amq-online-1-controller-manager) (Click "View All Tags" -> "1.1" -> Current 1.1 tag points to 1.1-5). 

This pull requests updates the packaging stucture versioning to the image tag version and also pull the 1.2 image.

Associated version bump in the integreatly operator - https://github.com/integr8ly/integreatly-operator/pull/133

## Verification
* Create a operator source using that points to my app registry:
```
apiVersion: operators.coreos.com/v1
kind: OperatorSource
metadata:
  name: integreatly-operators
  namespace: openshift-marketplace
spec:
  authorizationToken: {}
  displayName: Integreatly Operators
  endpoint: 'https://quay.io/cnr'
  publisher: Integreatly Publisher
  registryNamespace: kevfan
  type: appregistry
```
* Browese to the operator hub and verifiy that the AMQ Online version is `1.2.2`
* Install Integreatly using the following Installation.cr as an example:
```
apiVersion: integreatly.org/v1alpha1
kind: Installation
metadata:
  name: example-installation
spec:
  type: workshop
  namespacePrefix: integreatly-
  routingSubdomain: apps.cluster-id.openshiftworkshop.com
  masterUrl: https://console-openshift-console.apps.cluster-id.openshiftworkshop.com
  apiUrl: https://api.cluster-id.openshiftworkshop.com
  selfSignedCerts: true
```
* Verify that the `integreatly-amq-online` namespace is created
* Verify that the `enmasse-operator` pod is created
* Verify that the amq online route is accessible
	* Note: Would need to wait for the user-sso operator to be completed before being able to load  the route

### CODE REVIEW

File: integreatly-amq-online/amq-online-1.1.16/amq-online.1.1.16.clusterserviceversion.yaml [hunk 1]
```diff
@@ -3,13 +3,13 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: amqonline.0.28.0
+  name: amqonline.1.1-16
```
Comment (maleck13) [inline integreatly-amq-online/amq-online-1.1.16/amq-online.1.1.16.clusterserviceversion.yaml]: I think we can change this to 1.2.2

File: integreatly-amq-online/amq-online-1.1.16/amq-online.1.1.16.clusterserviceversion.yaml [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
   namespace: placeholder
   annotations:
     categories: "Streaming & Messaging"
     certified: "false"
     description: Red Hat AMQ Online provides messaging as a managed service on OpenShift
-    containerImage: registry.redhat.io/amq7/amq-online-1-controller-manager:1.1
+    containerImage: registry.redhat.io/amq7/amq-online-1-controller-manager:1.1-16
```
Comment (maleck13) [inline integreatly-amq-online/amq-online-1.1.16/amq-online.1.1.16.clusterserviceversion.yaml]: We should probably go with the floating tag for now so ``` registry.redhat.io/amq7/amq-online-1-controller-manager:1.2```

File: integreatly-amq-online/amq-online-1.1.16/amq-online.1.1.16.clusterserviceversion.yaml [hunk 3]
```diff
@@ -334,7 +334,7 @@ spec:
               serviceAccountName: enmasse-operator
               containers:
               - name: controller
-                image: registry.redhat.io/amq7/amq-online-1-controller-manager:1.1
+                image: registry.redhat.io/amq7/amq-online-1-controller-manager:1.1-16
```
Comment (maleck13) [inline integreatly-amq-online/amq-online-1.1.16/amq-online.1.1.16.clusterserviceversion.yaml]: same here registry.redhat.io/amq7/amq-online-1-controller-manager:1.2

File: integreatly-amq-online/amq-online.package.yaml [hunk 4]
```diff
@@ -1,4 +1,4 @@
 packageName: integreatly-amq-online
 channels:
 - name: integreatly
-  currentCSV: amqonline.0.28.0
+  currentCSV: amqonline.1.1-16
```
Comment (maleck13) [inline integreatly-amq-online/amq-online.package.yaml]: so this would become ```1.2.2```

**Comment by KevFan** [comment]
@maleck13 Cool thanks, I've updated the PR to use the `1.2` floating tag and changed the versioning/package structure to `1.2.2`

**Comment by maleck13** [comment]
@KevFan do you have a cluster with this version installed. I can check it there and then merge

**Pull request merged**